frontend/src/types/InviteCreateOptions.ts

```diff
@@ -0,0 +1,6 @@
+export type InviteCreateOptions = {
+  bucketId?: string;
+  email: string;
+  expiresAt?: string;
+  objectId?: string;
+};
```

Comment:
If we wanted to be absolutely precise about this and represent the XOR relationship between bucketId and objectId, we could pull off something like what is described in this SO: <https://stackoverflow.com/a/53229567>